css/components/badge.css

```diff
@@ -0,0 +1,63 @@
+/* === css for badges === */
+
+/* === badge on icons css */
+.badge-container{
+    position: relative;
+    font-size: 1.5rem;
+}
+.badge{
+    font-size: var(--normal-font-size);
+    position: absolute;
+    top: -0.3rem;
+    right: -0.8rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: .6rem;
+    width: 1.1rem;
+    height: 1.1rem;
+    border: .1rem solid #fff;
+    border-radius: 50%;
+    color: #fff;
+    background-color: var(--primary-dark);
+}
+
+/*== css for badges on avatars == */
+.badge-online{
+    position: absolute;
+    bottom: 0.5rem;
+    right: 0.1rem;
+    height: .9rem;
+    width: .9rem;
+    background-color: green;
+    border: .1rem solid #fff;
+    border-radius: 50%;
+}
+.badge-offline{
+    position: absolute;
+    bottom: 0.5rem;
+    right: 0.1rem;
+    height: .9rem;
+    width: .9rem;
+    background-color: var(--danger);
+    border: .1rem solid #fff;
+    border-radius: 50%;
+}
+
+/*== css for badges on button ==*/
+.btn-badge{
+    font-size: var(--normal-font-size);
```

Comment:
exactly the same properties with the different class names btn-badge and badge.

Reply:
sure will change that